src/convertors/mem/mem-convertor.ts

```diff
@@ -0,0 +1,108 @@
+import {markdownToHtml} from 'helpers/markdown'
+import {validateNotes} from 'helpers/validate'
+
+import {getDailyDate, parseTimestamp} from './mem-helpers'
+import {MemConversionError, MemExport, MemExportNote} from './types'
+import {
+  ConvertedNote,
+  ConvertOptions,
+  Convertor,
+  ConvertResponse,
+  REFLECT_HOSTNAME,
+} from '../../types'
+
+export class MemConvertor implements Convertor {
+  graphId: string
+  linkHost: string
+
+  constructor({
+    graphId,
+    linkHost = REFLECT_HOSTNAME,
+  }: {
+    graphId: string
+    linkHost?: string
+  }) {
+    this.graphId = graphId
+    this.linkHost = linkHost
+  }
+
+  accept = {'application/json': ['.json']}
+
+  convert({data}: ConvertOptions): ConvertResponse {
+    // 1. Parse JSON
+    const exportNotes = this.parse(data)
+
+    // 2. Convert markdown to html
+    const notes = exportNotes.map((note) => this.convertNote(note))
+
+    return validateNotes(notes)
+  }
+
+  private convertNote(memNote: MemExportNote): ConvertedNote {
+    const {
+      html,
+      backlinks,
+      tags: markdownTags,
+    } = markdownToHtml(memNote.markdown, {
+      graphId: this.graphId,
+      linkHost: this.linkHost,
+    })
+
+    const dailyDate = getDailyDate(memNote)
+
+    const updatedAt = parseTimestamp(memNote.updated)
+    const createdAt = parseTimestamp(memNote.created)
+
+    // Combine unique tags from markdown and mem
+    const tags = [...new Set([...markdownTags, ...memNote.tags])]
```

Comment:
Let's slugify all note ids and tag, just in case that Mem's format doesn't fit our requirement

Reply:
[slugify](https://github.com/sindresorhus/slugify)

Reply:
<img width="802" alt="CleanShot 2023-05-08 at 10 13 03@2x" src="https://user-images.githubusercontent.com/2142/236847672-d2e3f4cc-f806-4290-9ccc-d4251bde1ba6.png">

Not sure this is a great idea.

Reply:
You can use the decamelize option in the lib

Reply:
The IDs look consistent to me. I think let's reserve Slugify for when we're using something like slugifying titles.